src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -307,6 +307,10 @@ public Command changeElevation(Distance heightLevel) {
         });
   }
 
+  public Trigger elevatorUpTrigger() {
+    return elevatorAtSetpoint(getL0Setpoint()).negate().debounce(0.2);
+  }
+
```

Comment:
We shouldn't use this trigger, I would use the elevatorAtCurrentSetpoint rather than just checking if the elevator is up at all.